dist/index.js

```diff
@@ -257,22 +257,22 @@ parcelRequire = (function(e, r, t, n) {
         ],
         PiJs: [
             function(require, module, exports) {
-                const e = require('./fetcher'),
+                const e = require('delay'),
```

Comment:
Is this file needed to be commited?

Reply:
Okay, so as of now parcel-bundler seems to have a problem in this bundled file (implicit declaration of `parcelRequire`), so if anyone builds and runs the utility it won't work unless he/she manually prepends let/var to the file. So I guess we push this file (I've updated this to be in a working condition) and mention this in the readme as well. 

I guess we need to do this manually until we come across some workaround. :fearful: